code/modules/surgery/bodyparts/species_parts/moth_bodyparts.dm

```diff
@@ -1,43 +1,31 @@
 /obj/item/bodypart/head/moth
-	icon = 'icons/mob/species/moth/bodyparts.dmi'
-	icon_state = "moth_head"
-	icon_static = 'icons/mob/species/moth/bodyparts.dmi'
+	icon_greyscale = 'icons/mob/species/moth/bodyparts.dmi'
 	limb_id = SPECIES_MOTH
+	uses_mutcolor = TRUE
 	is_dimorphic = FALSE
-	should_draw_greyscale = FALSE
```

Comment:
`icon` and `icon_state` have no reason to be deleted
use `should_draw_greyscale` over `uses_mutcolor`